lbz/lambdas/client.py

```diff
@@ -67,3 +54,53 @@ def invoke(
         error_message = f"Error response from {function_name} Lambda (op: {op}): {lambda_result}"
         logger.error(error_message, extra={"data": data, "response": response})
         return response
+
+    @classmethod
+    def request(
+        cls,
+        function_name: str,
+        method: str,
+        path: str,
+        path_params: dict | None = None,
+        query_params: dict | None = None,
+        body: dict | None = None,
+        headers: dict | None = None,
+    ) -> Response:
+        # TODO: consider raise on error if resp >= 400
+        event = APIGatewayEvent(
+            method=method,
+            resource_path=path,
+            path_params=path_params,
+            query_params=query_params,
+            body=body,
+            headers=headers,
+        )
+
+        response = cls._invoke(function_name, payload=event)
+
+        return Response(
+            response["body"],
+            headers=response["headers"],
+            status_code=response["statusCode"],
+            base64_encoded=response["isBase64Encoded"],
+        )
+
+    @classmethod
+    def _invoke(cls, function_name: str, payload: dict, asynchronous: bool = False) -> dict:
+        raw_response = client.lambda_.invoke(
+            FunctionName=function_name,
+            Payload=json.dumps(payload, cls=cls.json_encoder).encode("utf-8"),
+            InvocationType="Event" if asynchronous else "RequestResponse",
+        )
+        if asynchronous:
```

Comment:
I would keep the original white line here:
```suggestion

        if asynchronous:
```